lib/tasks/add_user.rake

```diff
@@ -0,0 +1,14 @@
+namespace :user do
+  desc 'add user to database'
+
+  task :add, [:email, :first_name, :last_name, :role] => [:environment] do |t, args|
+    user = User.find_or_initialize_by(email: args[:email])
+    user.update(
```

Comment:
can we use `update!` here, as otherwise if it fails we won't know why. :)

Reply:
updated